internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -535,6 +566,42 @@ if ($with_fluentd) {
     echo "- Started"
 }
 
+if ($with_dotnet_instrumentation) {
+    echo "Installing SignalFx Instrumentation for .NET ..."
+    $api = "https://api.github.com/repos/signalfx/signalfx-dotnet-tracing/releases/latest"
+    $module_name = "Splunk.SignalFx.DotNet.psm1"
+    echo "Downloading .NET Instrumentation installer ..."
+    $download = (Invoke-WebRequest $api | ConvertFrom-Json).assets | Where-Object { $_.name -like $module_name } | Select-Object -Property browser_download_url,name
+    $dotnet_auto_path = Join-Path $env:temp $download.name
+    Invoke-WebRequest -Uri $download.browser_download_url -OutFile $dotnet_auto_path
+    Import-Module $dotnet_auto_path
+    echo "Installing SignalFx Dotnet Auto Instrumentation..."
+    Install-SignalFxDotnet
+
+    echo "Setting environment variables for instrumentation ..."
+    update_registry -path "$regkey" -name "COR_ENABLE_PROFILING" -value "1"
+    update_registry -path "$regkey" -name "COR_PROFILER" -value "{B4C89B0F-9908-4F73-9F59-0D77C5A06874}"
+    update_registry -path "$regkey" -name "CORECLR_ENABLE_PROFILING" -value "1"
+    update_registry -path "$regkey" -name "CORECLR_PROFILER" -value "{B4C89B0F-9908-4F73-9F59-0D77C5A06874}"
+
+    if ($instrumentation_exclude_processes -ne "") {
+      echo "Setting SIGNALFX_PROFILER_EXCLUDE_PROCESSES environment variable to $instrumentation_exclude_processes ..."
+      update_registry -path "$regkey" -name "SIGNALFX_PROFILER_EXCLUDE_PROCESSES" -value "$instrumentation_exclude_processes"
```

Comment:
When SignalFx is enabled globally, at least PowerShell must be definitely disabled. I would also add IIS manager and Server manager to the exclusion list. (these 3 are known not to be instrumented ever, but PowerShell is especially bad. If remote server is managed over remote PowerShell session, it will crash the connection and server is never accessible. At least one SWAT connected to that issue.)